challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,17 @@
+const binarySearch = (arr, searchTerm) => {
+  let startIndex = 0;
+  let endIndex = arr.length - 1;
+  while(startIndex <= endIndex) {
```

Comment:
just wondering what would happen if the arr wast a sorted number array, if this would work. i think you could write a much simpler for loop that was easier to read. but good job getting this to work.